src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -45,6 +45,26 @@
                     "type": "result"
                 }
             ]
+        },
+        {
+            "name": "ssm_publishDocument_create",
+            "description": "An SSM Document is created remotely",
```

Comment:
A or An?

Reply:
"An": https://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html

Reply:
Why we have 'a' on the other string 'Publish a SSM Document'

Reply:
We probably shouldn't :)

Reply:
This metric will be removed, and will double check for other grammar issues